### PR TITLE
feat: support --example-path-prefix

### PR DESCRIPTION
**Pull request by chalin**
cc @filiph @kwalrath

### CODE REVIEW

**Comment by kwalrath** [comment]
@devoncarew can you merge this?

**Comment by devoncarew** [comment]
Hey - in and out on vacation, but will review shortly.

File: bin/dartdoc.dart [hunk 1]
```diff
@@ -199,6 +200,8 @@ ArgParser _createArgsParser() {
   parser.addOption('hosted-url',
       help:
           'URL where the docs will be hosted (used to generate the sitemap).');
+  parser.addOption('example-path-prefix',
+      help: 'Prefix for @example paths. Default is project root.');
```
Comment (devoncarew) [inline bin/dartdoc.dart]: Perhaps `help: 'Prefix for @example paths.\n(defaults to the project root)'`, to match the output for other options.

**Comment by devoncarew** [comment]
lgtm; some minor wording change. I'll update that after merging.

**Comment by chalin** [comment]
Great, thanks!

**Pull request merged**